controller/controller.go

```diff
@@ -99,7 +99,7 @@ func New(config Config) (*Controller, error) {
 	}
 
 	if config.BackOffFactory == nil {
-		config.BackOffFactory = DefaultBackOffFactory()
+		config.BackOffFactory = func() backoff.Interface { return backoff.NewMaxRetries(7, 1*time.Second) }
```

Comment:
I like the addition of `NewMaxRetries` makes it more explicit how the backoff is being configured.

The number of retries and interval is repeated a lot but for me it makes the code easier to read. So I'm fine with it.

Reply:
You are right. Lets see how this goes and where we can find more common ground to abstract. I had a lot to do with using the abstracted library. We can iterate here.